.fixtures.yml

```diff
@@ -1,13 +1,8 @@
 fixtures:
   forge_modules:
-    apt:
-      repo: puppetlabs/apt
-      ref: 2.0.0
-    elastic_stack:
-      repo: elastic/elastic_stack
-      ref: 6.1.0
-    stdlib:
-      repo: puppetlabs/stdlib
-      ref: 4.21.0
+    apt: puppetlabs/apt
```

Comment:
can you please update this to git urls? we always test against master for unit tests and latest forge release in acceptance tests.

Reply:
Done! Will do the same for the other ES modules.